### ISSUE TITLE
Command based suggestion

### ISSUE THREAD

**Issue by Yeahhahaha** [Jisagi/Discord-guild-copy#49]
It would be epic if theres a way to make this command based. For example a certain user id can use these commands in a guild to backup, restore and clone.

**Comment by Jisagi** [comment]
It wouldn't be the first time thinking about porting the script to a bot but I decided against it and I'll tell you why.
1. There are already bots out there which do exactly what my script does. Which is in and of itself not a huge problem, since a bit more variety wouldn't be bad. One example would be a bot called [Xenon](https://discordbots.org/bot/xenon).
2. Discord themself didn't like the Bot Xenon and asked the developer for one single change. The bot allowed to restore guilds to a new guild id. In practical terms this means, that a guild that got deleted by discord for any kind of violation could be restored with ease to a newly created one. Discord obviously doesn't want that.

My script can do that as well, not because I advocate for such behaviour, but because I actually never thought about it when first writing it. As a bot, I could easily add a check to block this, just as Xenon did, but in the script version, it could be circumvented by just removing the few lines of code.

TL;DR: I don't want to write a bot with less features than the script version. My script was always intended for those people, who want to automate their backups the way they want, not inside a discord channel/guild, but (completely) external.

**Comment by Yeahhahaha** [comment]
> It wouldn't be the first time thinking about porting the script to a bot but I decided against it and I'll tell you why.
> 
> 1. There are already bots out there which do exactly what my script does. Which is in and of itself not a huge problem, since a bit more variety wouldn't be bad. One example would be a bot called [Xenon](https://discordbots.org/bot/xenon).
> 2. Discord themself didn't like the Bot Xenon and asked the developer for one single change. The bot allowed to restore guilds to a new guild id. In practical terms this means, that a guild that got deleted by discord for any kind of violation could be restored with ease to a newly created one. Discord obviously doesn't want that.
> 
> My script can do that as well, not because I advocate for such behaviour, but because I actually never thought about it when first writing it. As a bot, I could easily add a check to block this, just as Xenon did, but in the script version, it could be circumvented by just removing the few lines of code.
> 
> TL;DR: I don't want to write a bot with less features than the script version. My script was always intended for those people, who want to automate their backups the way they want, not inside a discord channel/guild, but (completely) external.

Is there any way I can make it command based, it would make it easier for me personally but I'm not confident in changing anything.

**Comment by Jisagi** [comment]
You would need to port the `copy.js` into a command like class to accept commandos and then just provide the necessary details for the other already written funtions like backup, restore, etc.
